aspnet-contrib/generator-aspnet-oauth
# Add initial generator

To test the generator:
- Ensure you have Yeoman installed
- Clone repo to your local computer
- CD into folder of repo and run `npm link`
- Now CD to the `/src` folder of your AspNet.Security.OAuth.Providers repo
- Run `yo aspnet-oauth` and complete the information

You should have a new folder with the project created. You can simply add project to solution, add reference to test project and all should be fine.

If you are happy with the code it generates and you have accepted this PR, then we can register it with the NPM registry so other people can grab it.

I will then also do a short video and to demonstrate to people how to use it and then I will base the CONTRIBUTING guide off of that.
